src/test/java/nl/beehivejava/scriptj/DefaultPluginManagerTest.java

```diff
@@ -0,0 +1,12 @@
+package nl.beehivejava.scriptj;
+
+/**
+ * @author Lesley
+ */
+public final class DefaultPluginManagerTest extends PluginManagerTest {
```

Comment:
![BLOCKER](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-blocker.png 'Severity: BLOCKER') Add some tests to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.tehmessiah.nl/coding_rules#rule_key=squid%3AS2187)